app/models/content_store_payload_version.rb

```diff
@@ -0,0 +1,41 @@
+class ContentStorePayloadVersion < ActiveRecord::Base
+  def self.increment(content_item_id:)
```

Comment:
Actually the more I think about this, `increment` may be a misnomer. Would `next` be more suitable? Given it's something resembling a sequence.